src/Appwrite/Platform/Modules/FunctionsVariables/Http/Create.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Appwrite\Platform\Modules\FunctionsVariables\Http;
+
+use Appwrite\Extend\Exception;
+use Appwrite\Utopia\Response;
+use Utopia\Database\Database;
+use Utopia\Database\DateTime;
+use Utopia\Database\Document;
+use Utopia\Database\Exception\Duplicate as DuplicateException;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Permission;
+use Utopia\Database\Helpers\Role;
+use Utopia\Database\Validator\Authorization;
+use Utopia\Database\Validator\UID;
+use Utopia\Platform\Action;
+use Utopia\Platform\Scope\HTTP;
+use Utopia\Validator\Text;
+
+class Create extends Action
+{
+    use HTTP;
+    public static function getName()
+    {
+        return 'create';
+    }
+
+    public function __construct()
+    {
+        $this
+            ->setHttpMethod(Action::HTTP_REQUEST_METHOD_POST)
+            ->setHttpPath('/v1/functions/:functionId/variables')
+            ->desc('Create variable')
+            ->groups(['api', 'functions'])
+            ->label('scope', 'functions.write')
+            ->label('audits.event', 'variable.create')
+            ->label('audits.resource', 'function/{request.functionId}')
+            ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
+            ->label('sdk.namespace', 'functions')
+            ->label('sdk.method', 'createVariable')
+            ->label('sdk.description', '/docs/references/functions/create-variable.md')
+            ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
+            ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+            ->label('sdk.response.model', Response::MODEL_VARIABLE)
+            ->param('functionId', '', new UID(), 'Function unique ID.', false)
+            ->param('key', null, new Text(Database::LENGTH_KEY), 'Variable key. Max length: ' . Database::LENGTH_KEY  . ' chars.', false)
+            ->param('value', null, new Text(8192, 0), 'Variable value. Max length: 8192 chars.', false)
+            ->inject('response')
+            ->inject('dbForProject')
+            ->inject('dbForConsole')
+            ->callback(fn ($functionId, $key, $value, $response, $dbForProject, $dbForConsole) => $this->action($functionId, $key, $value, $response, $dbForProject, $dbForConsole));
```

Comment:
can we check and see if this works?

```suggestion
            ->callback([$this, 'action']);
```